scss/variables/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$badge-height: 120px;
+$badge-width: 95px;
+$book-cover-height: 365px;
+$book-cover-width: 260pxx;
+$book-cover-list-height: 200px;
+$book-cover-list-width: 142px;
+$book-container-height: 415px;
+$book-container-width: 900px;
+$book-detail-font-size: 20px;
```

Comment:
Maybe you wanna use this values in another places, so the name would be more generic like _$regular-font-size_

Reply:
Also, isn't necessary to put everything into variables :laughing: